anime_downloader/extractors/kwik.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import re
+from bs4 import BeautifulSoup
+
+from anime_downloader.extractors.base_extractor import BaseExtractor
+from anime_downloader import session
+
+session = session.get_session()
+
+
+class Kwik(BaseExtractor):
+    '''Extracts video url from mp4upload embed pages, performs a request
```

Comment:
This comment needs to be changed or omitted since it is the same as the MP4Upload one.

Reply:
Yup Thanks for review, I copied and modified the mp4upload some lingering comments persist. 
I will update